tools/webpack/packages.js

```diff
@@ -142,6 +142,14 @@ module.exports = {
 		devtoolNamespace: 'wp',
 		filename: './build/[name]/index.min.js',
 		path: join( __dirname, '..', '..' ),
+		devtoolModuleFilenameTemplate: ( info ) => {
+			if ( info.resourcePath.includes( '@wordpress' ) ) {
+				const resourcePath =
+					info.resourcePath.split( '@wordpress' )[ 1 ];
+				return `../../packages/${ resourcePath }`;
+			}
+			return `webpack://${ info.namespace }/${ info.resourcePath }`;
+		},
```

Comment:
I noticed that I have a duplicate slash.

![double-border](https://github.com/WordPress/gutenberg/assets/54422211/6f76870d-7633-42d1-bb71-206dad738628)

Additionally, to check namespaces more strictly, how about the following change? Even on Windows, `info.resourcePath` had a forward slash instead of a backslash as the path separator, so I think this style is fine.

```suggestion
		devtoolModuleFilenameTemplate: ( info ) => {
			if ( info.resourcePath.includes( '/@wordpress/' ) ) {
				const resourcePath =
					info.resourcePath.split( '/@wordpress/' )[ 1 ];
				return `../../packages/${ resourcePath }`;
			}
			return `webpack://${ info.namespace }/${ info.resourcePath }`;
		},
```

Reply:
@t-hamano I missed it. Thank you !